controllers/RatingController.js

```diff
@@ -0,0 +1,46 @@
+import models from '../database/models';
+import ServerResponse from '../modules';
+
+const { successResponse, notFoundError } = ServerResponse;
+
+/**
+ * @class RatingController
+ * @exports RatingController
+ */
+export default class RatingController {
+  /**
+     *
+     * @method getArticleRatings
+     *
+     * @description get all ratings for an article
+     *
+     * @param {object} req express request object
+     * @param {object} res express response object
+     * @param {object} next express next object
+     *
+     * @returns {object} returns profile info
+     * @memberof RatingController
+     */
+  static async getArticleRatings(req, res, next) {
+    try {
+      const id = parseInt(req.params.id, 10);
```

Comment:
I think you'll need to fix parsing the id as an integer, because if my pr on uuid is merged this would break